spec/requests/self_assessment_pages_spec.rb

```diff
@@ -56,13 +56,21 @@
                                     text: 'These are some notes that I have written')
         fill_in 'Performance Assessment', with: 'Now I have edited my self-assessment.'
 
-
         click_button 'Save Changes'
         current_path.should == summary_review_path(review)
 
         self_assessment.reload
         self_assessment.response.should == 'Now I have edited my self-assessment.'
       end
+
+      it "doesn't let ACs submit blank self assessments" do
+        fill_in 'Performance Assessment', with: ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.